python/tk_framework_desktopserver/settings.py

```diff
@@ -140,6 +153,10 @@ def dump(self, logger):
         """
         Dumps all the settings into the logger.
         """
+        if self.integration_enabled is None:
```

Comment:
Why/How would this be the case? Assuming it's possible, should it be? The property implies that you should expect boolean return, which feels better to me than boolean or maybe None.